workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml

```diff
@@ -236,3 +236,22 @@ mapaction_rivers_length:
     Length of objects identified as rivers (or riverbanks)
   endpoint: elements/length
   filter: waterway in (riverbank,river) and type:way
+
+ideal_vgi_infrastructure:
```

Comment:
If I remember correctly Alex specifically requested to drop project-specific naming, if possible. We are happy to incorporate any definition of 'infrastructure' you come up with, as long as it is superior to this one, right Leonie? If you want to have separate infrastructure definitions I would suggest something like `infrastructure-full` with this definition and then e.g. `infrastructure-transportation` with only _highways_ and _railways_.